repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/CommandExecutor.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.lint
+
+import java.io.{File, InputStream, PrintStream}
+
+import com.typesafe.config.Config
+import org.openrepose.lint.commands.CommandRegistry
+import scopt.OptionParser
+
+/**
+  * A command line parser which determines what command to execute and executse it.
+  */
+object CommandExecutor {
+
+  def execute(in: InputStream, out: PrintStream, err: PrintStream, config: Config, args: Array[String]): Int = {
+    // Attach the actual console output streams to the passed in streams
+    Console.setOut(out)
+    Console.setIn(in)
+    Console.setErr(err)
+
+    val lintVer = config.getString("version")
+
+    val parser = new OptionParser[LintConfig]("repose-lint") {
+      head("repose-lint", lintVer)
+      opt[File]('c', "config-dir") valueName "<dir>" action { (x, c) =>
```

Comment:
maybe put in a line break between each option, and a comment saying what it is? then even though i can't read this gobblygook i at least know what they are